website/content/docs/templates/hcl_templates/functions/index.mdx

```diff
@@ -18,9 +18,9 @@ arguments in parentheses:
 max(5, 12, 9)
 ```
 
-For more details on syntax, see
-[_Function Calls_](/packer/docs/templates/hcl_templates/expressions#function-calls)
-on the Expressions page.
+For more details on syntax, see the [Expressions](/packer/docs/templates/hcl_templates/expressions) page, especially the
+[String Literals](https://developer.hashicorp.com/packer/docs/templates/hcl_templates/expressions#string-literals) section
+for information on how to invoke functions in string literals.
```

Comment:
Slight reword to use inclusive wording, and remove filler words. 
```suggestion
For more details on syntax, refer to the [Expressions](/packer/docs/templates/hcl_templates/expressions) page, especially the
[String Literals](https://developer.hashicorp.com/packer/docs/templates/hcl_templates/expressions#string-literals) section on invoking functions in string literals.
```

TBH the string literals and sting template sections could use some work. 

What do you think about changing this URL to the strings template section since it speaks more to using expressions in strings?